bolt-modules/boltlib/lib/puppet/functions/catch_errors.rb

```diff
@@ -12,14 +12,14 @@
   #         otherwise the result will be returned
```

Comment:
Should this also update `node` to `target` in the the spec directory here? Just while we're here:
```
lucy ~/githubs/bolt ((cf68954...)) $ ag -rl "node" bolt-modules/*
bolt-modules/boltlib/spec/functions/puppetdb_fact.rb
bolt-modules/boltlib/spec/functions/run_plan_spec.rb
bolt-modules/boltlib/spec/functions/run_command_spec.rb
bolt-modules/boltlib/spec/functions/puppetdb_query.rb
bolt-modules/boltlib/spec/functions/upload_file_spec.rb
bolt-modules/boltlib/spec/functions/run_task_spec.rb
bolt-modules/boltlib/spec/functions/apply_prep_spec.rb
bolt-modules/boltlib/spec/functions/run_script_spec.rb
bolt-modules/boltlib/spec/functions/get_resources_spec.rb
```

Reply:
Also update in `documentation/bolt_configuration_options.md`, `doc/index.html, and `doc/file.README.html`